test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +35,27 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void isAnyNull() {
+        // no items
+        assertFalse(Utils.isAnyNull());
+        
```

Comment:
There's trailing whitespace here, but that's fine for now. The real problem is that travis isn't checking for whitespace errors; When someone implements that it would be easily cleaned up along with the rest of the code base.

Reply:
This is not the only instance. Do check for trailing whitespace in the entire hunk.

Reply:
Thanks for the heads up. I've fixed them.

Reply:
> Thanks for the heads up. I've fixed them.

Ready for review? You'll need to submit a new version for that.

Reply:
Yup. Will do soon. Waiting for Travis to clear up...

Reply:
Yup that's a good thing to do :+1: